index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>landing page</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="js/index.js"></script>
+  </head>
+
+  <body>
+    <header id="header">
+      <h1>Loicard</h1>
+      <nav id="nav-bar">
+        <a href="#" id="nav-link">Home</a>
+        <a href="#" id="nav-link">About us</a>
+        <a href="#" id="nav-link">Contact Us</a>
+      </nav>
+    </header>
+
+    <div id="video">
+      <iframe
```

Comment:
- [] Kindly use An inline frame marked up as follow: <iframe src="...." title="..."></iframe>
-

Reply:
Kindly provide a reason why the student should do it this way instead of what he has done. This will help him/her to understand what you are requesting, hence will be easy to implement. :pray: